### PR TITLE
docs: update skip commit files docs

### PR DESCRIPTION
**Pull request by abeizn**
# Summary
docs: update skip commit files docs

### Does this close any open issues?
Closes na

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: docs/Plugins/gitextractor.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ This plugin extracts commits and references from a remote or local git repositor
    NOTE: you can run only one issue collection stage as described in the Github Plugin README.
 3. Use the [RefDiff](./refdiff.md) plugin to calculate version diff, which will be stored in `refs_commits` table.
 
+Note: If you do not want to collect commit files, you can bypass this step by setting the SKIP_COMMIT_FILES=true in environment variable or .env file. This will prevent the plugin from collecting commit file data.
```
Comment (Startrekzky) [inline docs/Plugins/gitextractor.md]: Please update the copywriting from 'in environment variable or .env file' to 'in the .env file'

File: docs/Troubleshooting/MySqlIssue.md [hunk 2]
```diff
@@ -100,4 +100,12 @@ services:
...
\ No newline at end of file
+   Note: in Python plugins, it will fallback to the `skip-verify` policy.
+
+### How to skip "subtask collectGitCommits ended unexpectedly, deadlock found when trying to get lock"? 
+When users have large repositories with a substantial number of commit files to collect, it can lead to the following issues:
+
+    1. The time to collect Git data is too long
+    2. Errors occurred while writing data to the DevLake database
+   
+To address this, you can bypass this step by setting the SKIP_COMMIT_FILES=true in environment variable or .env file.
```
Comment (Startrekzky) [inline docs/Troubleshooting/MySqlIssue.md]: Same as above

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: docs/Troubleshooting/MySqlIssue.md (hunk 2 above)
Comment (abeizn) [reply docs/Troubleshooting/MySqlIssue.md]: done

File: docs/Plugins/gitextractor.md (hunk 1 above)
Comment (abeizn) [reply docs/Plugins/gitextractor.md]: done

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**